webapp/src/index.tsx

```diff
@@ -1,6 +1,6 @@
 import {Store, Action} from 'redux';
 
-import {GlobalState} from 'mattermost-redux/types/store';
+import {GlobalState} from 'mattermost-webapp/packages/mattermost-redux/src/types/store';
```

Comment:
Are we able to successfully import and use functional exports from `mattermost-redux`, such as selectors?

Reply:
I just pushed a commit that allows for this to be imported that way. Had to figure out all the alias mess with eslint, webpack, and typescript.